EToCalculator.ts

```diff
@@ -0,0 +1,62 @@
+import { EToData } from "./types";
+
+// The implementation of this algorithm was guided by a step-by-step breakdown (http://edis.ifas.ufl.edu/pdffiles/ae/ae45900.pdf)
+/**
+ * Calculates the reference evapotranspiration using the Penman-Monteith (FAO-56) method (http://www.fao.org/3/X0490E/x0490e07.htm).
+ *
+ * @param etoData The data to calculate the ETo with.
+ * @param elevation The elevation above sea level of the watering site (in meters).
+ * @return The reference evapotranspiration (in millimeters per day).
+ */
+export function calculateETo( etoData: EToData, elevation: number ): number {
+	const avgTemp = ( etoData.maxTemp + etoData.minTemp ) / 2;
+
+	// Adjust the wind speed to a 2m height.
+	const windSpeed = etoData.windSpeed * 4.87 / Math.log( 67.8 * etoData.windSpeedMeasurementHeight - 5.42 );
+
+	const saturationVaporPressureCurveSlope = 4098 * 0.6108 * Math.exp( 17.27 * avgTemp / ( avgTemp + 237.3 ) ) / Math.pow( avgTemp + 237.3, 2 );
+
+	const pressure = 101.3 * Math.pow( ( 293 - 0.0065 * elevation ) / 293, 2.36 );
```

Comment:
Should this be raised to the power 5.26 rather than 2.36 ? Comparing with [Step 5](http://edis.ifas.ufl.edu/pdffiles/ae/ae45900.pdf)

Reply:
Yes it should, I have no idea where `2.36` came from. Nice catch!